_posts/en/newsletters/2021-08-11-newsletter.md

```diff
@@ -0,0 +1,168 @@
+  a2link="https://bitcoincore.reviews/22383#l-56"
+
+  q3="If we are looking up the transaction using the txindex, what are the steps? How much data is
+  deserialized?"
+  a3="The txindex maps from transaction id to the file, block position (similar to the block index),
+  and the offset within the blk\*.dat file where the transaction starts. We fetch and deserialize the
+  block header and transaction. The header is 80B and allows us to return the block hash to the user
+  (which is information not stored in the txindex). The transaction
+  can be any size but is typically thousands of times smaller than the block."
+  a3link="https://bitcoincore.reviews/22383#l-88"
+
+  q4="The first version of this PR included a behavior change: when an incorrect `block_index` is
+  provided to `GetTransaction`, find and return the tx anyway using the txindex. Do you think this change is an improvement,
+  and should it be included in this PR?"
+  a4="Participants agreed that it could be helpful but misleading, and that notifying the
+  user of the incorrect block hash input would be better. They also noted that a performance
+  improvement and behavior change would be best split into separate PRs."
+  a4link="https://bitcoincore.reviews/22383#l-128"
+%}
+
+## Preparing for taproot #8: multisignature nonces
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/07-nonces.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.1][] is a release
+  that contains a number of new features, several bug fixes,
+  and a few updates to developing protocols (including [dual
+  funding][topic dual funding] and [offers][topic offers]).
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  and enhanced support for hardware wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21528][] aims to improve the propagation of addresses
```

Comment:
Given that "addresses" is mostly used in the context of defining recipient scriptPubKeys, it would maybe be good to qualify what sort of addresses we are talking about here. ("node" doesn't feel amazing here, but I haven't been able to think of something better yet.)

```suggestion
- [Bitcoin Core #21528][] aims to improve the propagation of node addresses
```